keyboards/rama/u80_a/readme.md

```diff
@@ -0,0 +1,15 @@
+# RAMA U80-A
+
+![RAMA U80-A](https://something.com/something.jpg)
+
+A TKL keyboard. [More info at RAMA WORKS](https://rama.works/#/tkl-a/)
+
+Keyboard Maintainer: [Wilba6582](https://github.com/Wilba6582)  
+Hardware Supported: RAMA U80-A PCB  
+Hardware Availability: [RAMA WORKS Store](https://ramaworks.store/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make rama/u80_a:default
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
Please replace this line with the following:

`See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).`